slitlessutils/core/preprocess/background/background.py

```diff
@@ -364,6 +371,60 @@ def master(self, sci, hdr, unc, mod, gpx, img):
 
         return out, np.logical_not(sky)
 
+    @background_processing(mastersky=False)
+    def constant(self, sci, hdr, unc, mod, gpx):
+        r"""
+        Function to fit a constant background to a WFSS image, which
+        is taken as a sigma clipped median.
+
+        Parameters
+        ----------
+        data : str or `wfss.WFSS`
+           Either a string that is the full path to a WFSS file or a
+           `wfss.WFSS`.
+
+        Returns
+        -------
+        outfile : str
+           The name of the master-sky subtracted file.
+        """
+        # find the initial sky pixels
+        sky = self.skypixels(sci, unc, mod)
+        npix = np.count_nonzero(sky)
+
+        # initialize the iterations
+        proceed = True
+        it = 1
+        while proceed:
+
+            # find the good pixels
+            g = np.logical_and(sky, gpx)
+
+            # compute the 'average'
+            ave, med, sig = sigma_clipped_stats(sci[g], sigma=self.skysigma)
+
+            # re-find sky pixels
+            sky = self.skypixels(sci, unc, med)
+
+            # update the iteration values
+            it += 1
+            npix2 = np.count_nonzero(sky)
+            proceed = (npix != npix2) and (it < self.maxiter)
+            npix = npix2
+            if not proceed:
+                break
```

Comment:
```suggestion
```
You don't need to manually break a while loop if the continuation condition is False.